src/Common/tests/System/Diagnostics/RemoteExecutorConsoleApp/project.json

```diff
@@ -1,12 +1,12 @@
 {
   "dependencies": {
-    "System.Runtime": "4.1.1-beta-24411-04",
-    "Microsoft.NETCore.Platforms": "1.0.2-beta-24411-04",
+    "System.Runtime": "4.1.1-beta-24412-02",
+    "Microsoft.NETCore.Platforms": "1.0.2-beta-24412-02",
     "System.Console": "4.0.0",
```

Comment:
Didn't see this comment because it's not "me" that made this PR. :smile: Yeah, for project.json upgrades, the default is to not touch stable dependencies when upgrading. `System.Console` is in the buildinfo on the versions repo so it's auto-updated when the dependency is prerelease.